### ISSUE TITLE
[Label Data] Add AI label data

### ISSUE THREAD

**Issue by frank-zsy** [X-lab2017/open-digger#1195]
### Label

Artificial Intelligence

### Type

Techology

### Repos/Orgs/Users

- tensorflow/tensorflow
- kimiyoung/transformer-xl
- zihangdai/xlnet
- tensorflow/tensorboard
- lanpa/tensorboardX
- pytorch/xla
- google/jax
- google/deepdream
- google-research/t5x
- tensorflow/adanet
- google/dopamine
- google/model_search
- kubeflow/kubeflow
- google-research/tensor2robot
- pytorch/pytorch
- facebookresearch/detectron2
- facebookresearch/DensePose
- facebookresearch/fastText
- facebookresearch/pytorch3d
- facebookresearch/faiss
- facebookresearch/dlrm
- facebookresearch/pytext
- keras-team/keras
- onnx/onnx
- chainer/chainer
- torch/torch7
- BVLC/caffe
- facebookarchive/caffe2
- deepmind/alphafold
- deepmind/lab
- deepmind/trfl
- deepmind/open_spiel
- deepmind/pysc2
- deepmind/bsuite
- nltk/nltk
- microsoft/InnerEye-DeepLearning
- microsoft/onnxruntime
- PipedreamHQ/pipedream
- microsoft/NeuronBlocks
- interpretml/interpret
- microsoft/CNTK
- microsoft/nni
- microsoft/DeepSpeed
- microsoft/malmo
- Azure/counterfit
- apache/mxnet
- apache/tvm
- apache/mahout
- apache/opennlp
- apache/systemds
- apache/predictionio
- Theano/Theano
- autogluon/autogluon
- neo-ai/neo-ai-dlr
- openai/gym
- scikit-learn/scikit-learn
- flairNLP/flair
- Artelnics/opennn
- NVIDIA/vid2vid
- NVIDIA/TensorRT
- NVIDIA/DALI
- NVIDIAGameWorks/kaolin
- NVlabs/SPADE
- NVIDIA/Megatron-LM
- SharpAI/DeepCamera
- flyteorg/flyte
- lutzroeder/netron
- RaRe-Technologies/gensim
- statsmodels/statsmodels
- horovod/horovod
- uber/manifold
- stanfordnlp/CoreNLP
- deeplearning4j/deeplearning4j
- numpy/numpy
- Netflix/metaflow
- ultralytics/yolov3
- eriklindernoren/PyTorch-YOLOv3
- openvinotoolkit/openvino
- MycroftAI/mycroft-core
- opencv/opencv
- PaddlePaddle/Paddle
- PaddlePaddle/PaddleOCR
- PaddlePaddle/PaddleDetection
- PaddlePaddle/PaddleHub
- PaddlePaddle/PaddleNLP
- PaddlePaddle/PaddleSeg
- PaddlePaddle/PaddleSpeech
- PaddlePaddle/PaddleGAN
- PaddlePaddle/PaddleClas
- PaddlePaddle/PaddleX
- PaddlePaddle/PaddleRec
- PaddlePaddle/Paddle-Lite
- PaddlePaddle/PaddleSlim
- PaddlePaddle/PaddleVideo
- PaddlePaddle/ERNIE
- h2oai/h2o-3
- explosion/spaCy
- Jittor/jittor
- MegEngine/MegEngine
- Oneflow-Inc/oneflow
- alibaba/MNN
- alibaba/euler
- alibaba/EasyNLP
- alibaba/EasyRec
- alibaba/EasyCV
- alibaba/EasyTransfer
- sql-machine-learning/elasticdl
- sql-machine-learning/sqlflow
- mindspore-ai/mindspore
- ApolloAuto/apollo
- XiaoMi/mace
- kaldi-asr/kaldi
- bytedance/byteps
- bytedance/fedlearner
- Angel-ML/angel
- Tencent/ncnn
- Tencent/PocketFlow
- Tencent/plato
- Qihoo360/XLearning
- milvus-io/milvus
- lancopku/pkuseg-python
- open-mmlab/mmdetection
- open-mmlab/mmcv
- open-mmlab/mmsegmentation
- open-mmlab/mmpose
- open-mmlab/mmocr
- open-mmlab/mmclassification
- open-mmlab/mmdetection3d
- open-mmlab/mmaction2
- open-mmlab/mmediting
- open-mmlab/mmrotate
- seetaface/SeetaFaceEngine
- didi/AoE
- delta-io/delta
- OAID/Tengine
- WeBankFinTech/Prophecis
- FederatedAI/FATE
- DeepWisdom/AutoDL
- hankcs/HanLP

The following company / region label data should also be added.

To google:

- kubeflow
- google-research

To deepmind:

- deepmind / America: 

To microsfot:

- interpretml
- Azure

To amazon:

- autogluon
- neo-ai

To openai / America:

- openai

To zalando / Germany:

- zalandoresearch
- flairNLP

To artelnics / Spain:

- Artelnics

To nvidia / America:

- NVIDIA
- NVIDIAGameWorks
- NVlabs

To rare technologies / United Kingdom

- RaRe-Technologies

To uber / America:

- uber
- uber-go
- horovod

To intel / America:

- intel
- IntelLabs
- openvinotoolkit

To MycroftAI / America:

- MycroftAI

To h2oai / America:

- h2oai

To explosionai / Germany:

- explosion

To huawei:

- mindspore-ai

To xiaomi:

- kaldi-asr

To tencent:

- Angel-ML

To webank:

- FederatedAI

To fuzhi / China:

- DeepWisdom

### Other information

The data is from the landscape of new Programmer magazine of CSDN.

**Comment by Gdakgdak22** [comment]
> ### Label
> Artificial Intelligence
> 
> ### Type
> Techology
> 
> ### Repos/Orgs/Users
> * tensorflow/tensorflow
> * kimiyoung/transformer-xl
> * zihangdai/xlnet
> * tensorflow/tensorboard
> * lanpa/tensorboardX
> * pytorch/xla
> * google/jax
> * google/deepdream
> * google-research/t5x
> * tensorflow/adanet
> * google/dopamine
> * google/model_search
> * kubeflow/kubeflow
> * google-research/tensor2robot
> * pytorch/pytorch
> * facebookresearch/detectron2
> * facebookresearch/DensePose
> * facebookresearch/fastText
> * facebookresearch/pytorch3d
> * facebookresearch/faiss
> * facebookresearch/dlrm
> * facebookresearch/pytext
> * keras-team/keras
> * onnx/onnx
> * chainer/chainer
> * torch/torch7
> * BVLC/caffe
> * facebookarchive/caffe2
> * deepmind/alphafold
> * deepmind/lab
> * deepmind/trfl
> * deepmind/open_spiel
> * deepmind/pysc2
> * deepmind/bsuite
> * nltk/nltk
> * microsoft/InnerEye-DeepLearning
> * microsoft/onnxruntime
> * PipedreamHQ/pipedream
> * microsoft/NeuronBlocks
> * interpretml/interpret
> * microsoft/CNTK
> * microsoft/nni
> * microsoft/DeepSpeed
> * microsoft/malmo
> * Azure/counterfit
> * apache/mxnet
> * apache/tvm
> * apache/mahout
> * apache/opennlp
> * apache/systemds
> * apache/predictionio
> * Theano/Theano
> * autogluon/autogluon
> * neo-ai/neo-ai-dlr
> * openai/gym
> * scikit-learn/scikit-learn
> * flairNLP/flair
> * Artelnics/opennn
> * NVIDIA/vid2vid
> * NVIDIA/TensorRT
> * NVIDIA/DALI
> * NVIDIAGameWorks/kaolin
> * NVlabs/SPADE
> * NVIDIA/Megatron-LM
> * SharpAI/DeepCamera
> * flyteorg/flyte
> * lutzroeder/netron
> * RaRe-Technologies/gensim
> * statsmodels/statsmodels
> * horovod/horovod
> * uber/manifold
> * stanfordnlp/CoreNLP
> * deeplearning4j/deeplearning4j
> * numpy/numpy
> * Netflix/metaflow
> * ultralytics/yolov3
> * eriklindernoren/PyTorch-YOLOv3
> * openvinotoolkit/openvino
> * MycroftAI/mycroft-core
> * opencv/opencv
> * PaddlePaddle/Paddle
> * PaddlePaddle/PaddleOCR
> * PaddlePaddle/PaddleDetection
> * PaddlePaddle/PaddleHub
> * PaddlePaddle/PaddleNLP
> * PaddlePaddle/PaddleSeg
> * PaddlePaddle/PaddleSpeech
> * PaddlePaddle/PaddleGAN
> * PaddlePaddle/PaddleClas
> * PaddlePaddle/PaddleX
> * PaddlePaddle/PaddleRec
> * PaddlePaddle/Paddle-Lite
> * PaddlePaddle/PaddleSlim
> * PaddlePaddle/PaddleVideo
> * PaddlePaddle/ERNIE
> * h2oai/h2o-3
> * explosion/spaCy
> * Jittor/jittor
> * MegEngine/MegEngine
> * Oneflow-Inc/oneflow
> * alibaba/MNN
> * alibaba/euler
> * alibaba/EasyNLP
> * alibaba/EasyRec
> * alibaba/EasyCV
> * alibaba/EasyTransfer
> * sql-machine-learning/elasticdl
> * sql-machine-learning/sqlflow
> * mindspore-ai/mindspore
> * ApolloAuto/apollo
> * XiaoMi/mace
> * kaldi-asr/kaldi
> * bytedance/byteps
> * bytedance/fedlearner
> * Angel-ML/angel
> * Tencent/ncnn
> * Tencent/PocketFlow
> * Tencent/plato
> * Qihoo360/XLearning
> * milvus-io/milvus
> * lancopku/pkuseg-python
> * open-mmlab/mmdetection
> * open-mmlab/mmcv
> * open-mmlab/mmsegmentation
> * open-mmlab/mmpose
> * open-mmlab/mmocr
> * open-mmlab/mmclassification
> * open-mmlab/mmdetection3d
> * open-mmlab/mmaction2
> * open-mmlab/mmediting
> * open-mmlab/mmrotate
> * seetaface/SeetaFaceEngine
> * didi/AoE
> * delta-io/delta
> * OAID/Tengine
> * WeBankFinTech/Prophecis
> * FederatedAI/FATE
> * DeepWisdom/AutoDL
> * hankcs/HanLP
> 
> The following company / region label data should also be added.
> 
> To google:
> 
> * kubeflow
> * google-research
> 
> To deepmind:
> 
> * deepmind / America:
> 
> To microsfot:
> 
> * interpretml
> * Azure
> 
> To amazon:
> 
> * autogluon
> * neo-ai
> 
> To openai / America:
> 
> * openai
> 
> To zalando / Germany:
> 
> * zalandoresearch
> * flairNLP
> 
> To artelnics / Spain:
> 
> * Artelnics
> 
> To nvidia / America:
> 
> * NVIDIA
> * NVIDIAGameWorks
> * NVlabs
> 
> To rare technologies / United Kingdom
> 
> * RaRe-Technologies
> 
> To uber / America:
> 
> * uber
> * uber-go
> * horovod
> 
> To intel / America:
> 
> * intel
> * IntelLabs
> * openvinotoolkit
> 
> To MycroftAI / America:
> 
> * MycroftAI
> 
> To h2oai / America:
> 
> * h2oai
> 
> To explosionai / Germany:
> 
> * explosion
> 
> To huawei:
> 
> * mindspore-ai
> 
> To xiaomi:
> 
> * kaldi-asr
> 
> To tencent:
> 
> * Angel-ML
> 
> To webank:
> 
> * FederatedAI
> 
> To fuzhi / China:
> 
> * DeepWisdom
> 
> ### Other information
> The data is from the landscape of new Programmer magazine of CSDN.
